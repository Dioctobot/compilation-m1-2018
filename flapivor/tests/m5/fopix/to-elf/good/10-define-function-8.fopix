def f(x, y, z, t, u, v, w) =
  print_int (x);
  print_int (y);
  print_int (z);
  print_int (t);
  print_int (u);
  print_int (v);
  print_int (w)

val main = f (37, 73, 13, 31, 42, 91, 19)