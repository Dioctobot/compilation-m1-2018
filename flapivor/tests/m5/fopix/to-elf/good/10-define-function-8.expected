Process exited with status exited(0).
STDOUT:
37731331429119
STDERR:



